src/System.Private.ServiceModel/tools/dir.props

```diff
@@ -0,0 +1,8 @@
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
@StephenBonikowsky -- do additions of these files cause any concerns with the changes you are syncing from CoreFx?